client/src/components/CheckboxSetField/CheckboxSetField.js

```diff
@@ -132,6 +131,6 @@ CheckboxSetField.defaultProps = {
   value: [],
 };
 
-export { CheckboxSetField };
+export { CheckboxSetField as Component };
```

Comment:
yup, there's an eslint rule where `import { CheckboxField }`and `import CheckboxField` cannot both exist, which makes sense because it's easy to confuse between having the `{ }` wrappers and without it
So it made more sense to name the item export as `Component` here